samples/client/petstore/c/model/pet.c

```diff
@@ -110,6 +110,10 @@ cJSON *pet_convertToJSON(pet_t *pet) {
     {
         goto fail;
     }
+    if(cJSON_AddNumberToObject(photo_urls, "", *(double *)photo_urlsListEntry->data) == NULL)
+    {
+        goto fail;
+    }
```

Comment:
This sample code is not regenerated by the latest template code.

Reply:
Done, and pushed to github